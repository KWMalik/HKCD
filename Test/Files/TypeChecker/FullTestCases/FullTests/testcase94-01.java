//sample class 2
//Example with type errors and correct minijava syntax

class sample2{
	public static void main (String[] args){
	
		int ic = 1;
		int jc = 10;
		int kc = 0;
		int dsl = 0;
		
		kc = (ic + jc)/ic;
		dsl = kc * kc / ic / ic;
		
		System.out.println(kc * ic);
		System.out.println(kc);
				
	}
}

class sample2{			//source of error
	int j;
}