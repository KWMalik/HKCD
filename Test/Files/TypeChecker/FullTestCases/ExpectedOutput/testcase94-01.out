Class named sample2 already exists.
